tests/db/test_bigchain_api.py

```diff
@@ -1822,6 +1856,8 @@ def test_get_subcondition_from_vk(self, b, user_sk, user_vk):
             subcondition = condition.get_subcondition_from_vk(owner_after)[0]
             assert subcondition.public_key.to_ascii().decode() == owner_after
 
+    @pytest.mark.skipif(reason='@Dimi need explanation of what is actually'
+                        'going on here')
     @pytest.mark.usefixtures('inputs')
```

Comment:
Is there a reason why this isn't three separate tests? Is one of the `tx` reused? Something is becoming invalid along the way now.